### ISSUE TITLE
Suggestion: Multi-monitor support

### ISSUE THREAD

**Issue by Wimali** [OpenRCT2/OpenRCT2#2171]
I think there should be a way for the game to utilise multi-monitors in a smart way. For example, you could have a second (initially blank) monitor and be able to fill that with viewports, all kinds of windows (finances, research, people & people's thoughts, rides, etc.)

**Comment by pizza2004** [comment]
I agree.  Since it's possible using SDL2 to have multiple windows or whatever now this should technically be possible.  This would be a great addition to both OpenRCT2 and OpenTTD frankly.

**Comment by IntelOrca** [comment]
It would be a great deal of work to implement such a feature and work around all the complications. Lots of extra code would need to be written to manage multiple game windows and handle any kind of interface control between them. A great deal of interface changes would be required.

**Comment by pizza2004** [comment]
Yeah, it would be a feature for very far in the future for sure.  I could possibly look into it myself when the whole game is decompiled and such, but I'm probably not gonna touch the code very much until I know we know exactly what's going on with everything.

**Comment by jakeprem** [comment]
You could use the network protocols or something to export/expose data to a web interface. 

E.g. you could go to 127.0.0.1:8080 and see analytics on your park.

**Comment by PFCKrutonium** [comment]
The last time this was discussed, that is _literally_ what I suggested. The issue that was put forward was that it could be a potential issue if someone were to find an exploit in whatever webserver we would be using.

**Comment by pizza2004** [comment]
I think it would be better and more efficient to have it all in one program.  Some quick searches have told me it's entirely possible to have SDL2 make multiple fullscreen rendering surfaces.  So we could just open multiple 'windows' and have one be the main and the others be viewports.

The main thing to determine is what things can be done or opened and what interface shows up on the secondary windows.

**Comment by IntelOrca** [comment]
This is nothing to do with web servers or web interfaces... This is literally just allowing the game to display fullscreen at the correct resolution of every monitor - by using several windows.
